media/css/externalpages/pocket/components/_contact-info.scss

```diff
@@ -0,0 +1,37 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import './nav';
+@import './footer';
+@import '../utils/variables';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/base/elements/containers';
+
+.contact-info-header {
+    @include text-title-md;
+    font-family: $font-sans;
+    font-weight: 600;
+    margin-top: 0;
+}
+
+.contact-info-subheader {
+    @include text-title-2xs;
+    font-family: $font-sans;
+    font-weight: 600;
+}
+
+.contact-info-body-text {
+    color: $color-text-secondary;
+    font-family: $font-sans;
+    @include text-body-md;
+
+    a {
```

Comment:
None of the links on this page seem to have any hover/focus states which would be nice to add. Do we have a set of default link styles for Pocket pages that we could use here?